### PR TITLE
Feature/update requirements with hashes

### PR DESCRIPTION
**Pull request by gemmai95**
Adds hashes for each requirement this should help avoid accidental or malicious changes.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[Turn on PIP hashed requirements.txt (3)](https://trello.com/c/ZBaaoggQ/246-turn-on-pip-hashed-requirementstxt-3)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Unreleased
+  - Update requirements to include hashes
```
Comment (necrophonic) [inline CHANGELOG.md]: See comment on [sdx-rabbit-monitor](https://github.com/ONSdigital/sdx-rabbit-monitor/pull/29/files/c5002976f8e11a3b83c47a3304ce88dfecf8421f)

**Review by necrophonic — DISMISSED** [review]

**Review by tej99 — CHANGES_REQUESTED** [review]
"--require-hashes" missing from make file

**Review by tej99 — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 build:
 	git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
```
Comment (JamesGardiner) [inline Makefile]: Refs to sdx-common need to be removed

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

**Comment by gemmai95** [comment]
[Turn on PIP hashed requirements.txt (3)](https://trello.com/c/ZBaaoggQ/246-turn-on-pip-hashed-requirementstxt-3)

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**